### ISSUE TITLE
Not able to get it working with the latest flutter version - 3.13.7

### ISSUE THREAD

**Issue by rohanjsh** [gskinnerTeam/flutter-defer-pointer#9]
```dart
Widget build(BuildContext context) {
    return DeferredPointerHandler(
       child: SizedBox(
           width: 100,
           height: 100,
           child: Stack(clipBehavior: Clip.none, children: [
             // Hang button off the bottom of the content
             Positioned(
               bottom: -30,
               child: DeferPointer(child: _SomeBtn(false)),
             ),
             // Content
             Positioned.fill(
               child: Container(
                 decoration: BoxDecoration(color: Colors.green, boxShadow: [
                   BoxShadow(color: Colors.black.withOpacity(1), blurRadius: 4, spreadRadius: 4),
                 ]),
               ),
             ),
           ]))));
  }
```

When running the above sample code provided, I was not able to get it running on the latest flutter version.
 **No taps were detected.**

It'll be great if anyone can look into this.
Thanks

@gskinner

**Comment by vietstone-ng** [comment]
I've just tried with Flutter 3.13.8 and it works. Is your DeferredPointerHandler obscured (a part of it) by any outside widget?

**Comment by rohanjsh** [comment]
I don't think so, I tried running above example in my app but it did not detect any taps, it may be possible I am wrong somewhere.

For our usecase (with some complexity) widget tree looks something like this.
<img width="464" alt="Screenshot 2023-10-25 at 12 43 45" src="https://github.com/gskinnerTeam/flutter-defer-pointer/assets/35066779/d74f8522-963f-4099-b3dd-e52420419665">

Also, just to mention I tried using the `link` parameter as well, but it still didn't work.
